chassembly/src/main/java/org/quiltmc/chasm/lang/internal/parse/Parser.jj

```diff
@@ -29,7 +29,17 @@ import org.quiltmc.chasm.lang.api.ast.UnaryNode;
 import org.quiltmc.chasm.lang.internal.parse.SourceSpan;
 
 public class Parser {
-
+    private static String extractStringLiteral(String image) {
+        StringBuilder sb = new StringBuilder(image.length());
+        for (int i = 1, e = image.length() - 1; i < e; i++) {
```

Comment:
Why not?

Reply:
The for loop most people would expect would look `for (int i = 1; i < image.length() - 1; i++)`.
Since you didn't do that, surely there's a reason you did it different.
If not, then maybe doing it the "standard" way would be better.